autogpt_platform/frontend/src/app/signup/actions.ts

```diff
@@ -38,7 +38,7 @@ export async function signup(values: z.infer<typeof signupFormSchema>) {
       }
       console.log("Signed up");
```

Comment:
I know it's unrelated to the PR but can we remove it ?

Reply:
yes no problem